### PR TITLE
fix: [PIE-2907]: label map added for deployment types

### PR DESCRIPTION
**Pull request by bhavya-sinha**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by yc147 — DISMISSED** [review]

**Review by YounixIO — DISMISSED** [review]

**Review by shamala-harness — APPROVED** [review]

File: src/modules/70-pipeline/pages/pipelines/PipelinesPage.tsx [hunk 1]
```diff
@@ -364,7 +365,7 @@ function PipelinesPage({ mockData }: CDPipelinesPageProps): React.ReactElement {
   React.useEffect(() => {
     if (!isEmpty(deploymentTypeResponse?.data) && deploymentTypeResponse?.data) {
       const options: SelectOption[] = deploymentTypeResponse.data.map(type => ({
-        label: type === 'NativeHelm' ? getString('pipeline.nativeHelm') : getString('kubernetesText'),
+        label: getString(deploymentTypeLabel[type]),
```
[the remark this answers is not in the document]
Comment (bhavya-sinha) [reply src/modules/70-pipeline/pages/pipelines/PipelinesPage.tsx:368]: Ideally SSH should not be coming from the API. and that is already fixed.
We might have additional deployment types in future. So should we add filter on data?
Rather we can just check for type. but expectation is that type should always come.

**Review by samarth-sharma31 — APPROVED** [review]

**Comment by bhavya-sinha** [comment]
retrigger jest

**Comment by bhavya-sinha** [comment]
retrigger jest

**Review by samarth-sharma31 — APPROVED** [review]

**Comment by vkbansal** [comment]
Merging as unrelated test is failing

**Pull request merged**